versions/ufo3/lib.plist.md

```diff
@@ -97,6 +97,28 @@ The handling of the feature file is undefined.
 
 This data is optional. Glyph names must not occur more than once. The list may contain glyphs that are not in the font. An empty list or the absence of this key means that all glyphs are to be exported as-is with groups and kerning untouched.
 
+#### public.unicodeVariationSequences
+
+This key is a dictionary for Unicode Variation Sequences. Keys are *Variation Selector* code points as a hexadecimal numbers and values are dictionaries. These dictionaries contain code points as hexidecimal numbers as keys and glyph names as values. This data can be used to generate the [OpenType cmap subtable format 14] for Unicode [Variation Sequences].
```

Comment:
Based on the feedback we got, would be good to add that the tool that compiles the binary needs to decide which glyph is the default, this is only a mapping to feed information to a complier.

Reply:
I changed the following sentence:
```diff
- This data can be used to generate the [OpenType cmap subtable format 14] for Unicode [Variation Sequences].
+ This data and the glyphs' Unicode code points can be used to generate the [OpenType cmap subtable format 14] for Unicode [Variation Sequences].
```